res/GAMEHELP/GUMBALL

```diff
@@ -0,0 +1,25 @@
+
+ Sort gumballs, disarm bombs (level 4+)
+
+          joystick or keyboard
+
+       button 0 open gate, 1 to aim
+
+     Production       Aim (level 4+)
+
+         E              U I O
+
+       S D F            J K L?
```

Comment:
what does the question mark mean

Reply:
I haven't tested aim key controls myself. This was a reminder to test L to make sure it worked.